lib/trino-collect/src/test/java/io/trino/collect/cache/TestEvictableLoadingCache.java

```diff
@@ -372,4 +454,44 @@ public void testInvalidateAndLoadConcurrently(Invalidation invalidation)
             executor.awaitTermination(10, SECONDS);
         }
     }
+
+    /**
+     * A class that implements value-based equality taking into account some fields, but not all.
```

Comment:
The current wording breaks checkstyle rules (https://github.com/airlift/airbase/pull/312). Will workaround.

Reply:
(changed)